CHANGELOG.md

```diff
@@ -8,6 +8,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [vNext]
 ### Added
 - DDC staking initial impl in `ddc-staking` pallet
+- Testing utilities 
+- Benchmarking and autogenerated methods weights for ddc-staking pallet
```

Comment:
Use the same syntax formatting for `ddc-staking` as in #1

Reply:
updated